src/views/virtualmachines/details/VirtualMachinePendingChangesAlert/RestartPendingChanges.tsx

```diff
@@ -23,21 +24,21 @@ const RestartPendingChanges: React.FC<RestartPendingChangesProps> = ({
 }) => {
   const { t } = useKubevirtTranslation();
   const { featureEnabled: nicHotPlugEnabled } = useFeatures(BRIDGED_NIC_HOTPLUG_ENABLED);
-
-  const showRestartSection =
-    !nicHotPlugEnabled ||
-    nonHotPlugNICChangesExist(pendingChanges, hasPendingChange(nonHotPlugPendingChanges));
-
-  if (!showRestartSection) return null;
-
+  const pendingChangesTabs = getPendingChangesByTab(pendingChanges);
   const {
     pendingChangesDetailsTab,
     pendingChangesDisksTab,
     pendingChangesEnvTab,
     pendingChangesNICsTab,
     pendingChangesSchedulingTab,
     pendingChangesScriptsTab,
-  } = getPendingChangesByTab(pendingChanges);
+  } = pendingChangesTabs;
+  const showRestartSection =
+    !nicHotPlugEnabled ||
+    nonHotPlugNICChangesExist(pendingChanges, hasPendingChange(nonHotPlugPendingChanges)) ||
+    !isEmpty(Object.values(pendingChangesTabs).reduce((acc, val) => [...acc, ...val], []));
```

Comment:
PendingchangesTabs are arrays. I wound do:

```suggestion
  Object.values(pendingChangesTabs).some(changes => changes.length > 0)
```